app/shop.py

```diff
@@ -0,0 +1,23 @@
+class Shop:
+    def __init__(self, name: str, location: list, products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def print_receipt(self, customer: dict) -> None:
+        print(f"{customer.name} rides to {self.name}\n")
+        print(f"{"Date:"} 04/01/2021 {"12:33:41"}")

```

Comment:
The receipt formatting uses curly braces incorrectly. Consider using a different method for formatting strings, such as f-strings, to avoid syntax errors and improve readability.